lms/envs/common.py

```diff
@@ -408,6 +408,9 @@
 
     # Whether to display the account deletion section the account settings page
     'ENABLE_ACCOUNT_DELETION': True,
+
+    # Set this to manage the default value across all courses in a given deployment
+    'COURSE_DEFAULT_INVITE_ONLY': False,
```

Comment:
You should add some toggle annotations here. Take a look at https://github.com/edx/edx-platform/blob/master/lms/envs/common.py#L126-L137 for an example

I suggest:
```
    # .. toggle_name: COURSE_DEFAULT_INVITE_ONLY
    # .. toggle_type: feature_flag
    # .. toggle_default: False
    # .. toggle_description: Set this to manage the default value for INVITE_ONLY across all courses in a given deployment
    # .. toggle_category: admin
    # .. toggle_use_cases: open_edx
    # .. toggle_creation_date: 2019-05-16
    # .. toggle_expiration_date: None
    # .. toggle_tickets: https://github.com/mitodl/edx-platform/issues/123
    # .. toggle_status: unsupported
```

I omitted `toggle_warnings` because I don't think there are any. 

I'm not sure about the value for `toggle_category`